services/src/northbound/src/test/java/org/openkilda/northbound/utils/CorrelationIdPredictableFactory.java

```diff
@@ -0,0 +1,25 @@
+/* Copyright 2018 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.northbound.utils;
+
+public class CorrelationIdPredictableFactory extends CorrelationIdUuidFactory {
+    private int counter = 0;
```

Comment:
What's the purpose of state in spring singleton component? This is thread unsafe.